<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NameGenie - Batch File Renamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string><h1>NameGenie</h1></string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectFolderButton">
      <property name="text">
       <string>انتخاب پوشه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="prefixInput">
      <property name="placeholderText">
       <string>پیشوند (اختیاری)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="suffixInput">
      <property name="placeholderText">
       <string>پسوند (اختیاری)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ascendingCheckbox">
      <property name="text">
       <string>شماره‌گذاری صعودی</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="previewList">
      <property name="toolTip">
       <string>پیش‌نمایش تغییر نام فایل‌ها</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startRenamingButton">
      <property name="text">
       <string>تغییر نام فایل‌ها</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
